.travis/build_package.sh

```diff
@@ -67,35 +71,60 @@ do
   elif [ "$ARG" != Polyhedron_demo ]; then
     echo "No test found for $ARG"
 	fi
-	
-	if [ "$ARG" != Polyhedron ] && [ -d "$ROOT/$DEMO" ]
-	then
-	  cd $ROOT/$DEMO
-	  mkdir -p build
-	  cd build
-	  cmake -DCGAL_DIR="$ROOT/build" -DCMAKE_CXX_FLAGS_RELEASE="-DCGAL_NDEBUG" ..
-	  make -j2
-  elif [ "$ARG" != Polyhedron_demo ]; then
-    echo "No demo found for $ARG"
-	fi
-if [ "$ARG" == Polyhedron_demo ]; then
-  cd "$ROOT/Polyhedron/demo/Polyhedron"
-  #install libqglviewer
-  git clone --depth=1 https://github.com/GillesDebunne/libQGLViewer.git ./qglviewer
-  cd ./qglviewer/QGLViewer
-  #use qt5 instead of qt4
-  export QT_SELECT=5
-  qmake NO_QT_VERSION_SUFFIX=yes
-  make -j2
-  if [ ! -f libQGLViewer.so ]; then
-    echo "libQGLViewer.so not made"
-    exit 1
-  fi
-#end install qglviewer
-  cd "$ROOT/Polyhedron/demo/Polyhedron"
-  mkdir -p ./build
-  cd ./build
-  cmake -DCGAL_DIR="$ROOT/build" -DQGLVIEWER_INCLUDE_DIR="$ROOT/Polyhedron/demo/Polyhedron/qglviewer" -DQGLVIEWER_LIBRARIES="$ROOT/Polyhedron/demo/Polyhedron/qglviewer/QGLViewer/libQGLViewer.so" -DCMAKE_CXX_FLAGS_RELEASE="-DCGAL_NDEBUG" ..
-  make -j2
-fi
+  #Packages like Periodic_3_triangulation_3 contain multiple demos
+  for DEMO in $DEMOS; do
+	#If there is no demo subdir, try in GraphicsView
+    if [ ! -d $ROOT/$DEMO ] || [ ! -f "$ROOT/$DEMO/CmakeLists.txt" ]; then
+     DEMO="GraphicsView/demo/$ARG"
+    fi
+	  if [ "$ARG" != Polyhedron ] && [ -d "$ROOT/$DEMO" ]
+  	then
+      if [ $NEED_3D == 1 ]; then
```

Comment:
`==` is a bash-ism. The correct way to test strings equality in shell is a single `=`.